src/Sylius/Behat/Context/Setup/CurrencyContext.php

```diff
@@ -150,9 +150,6 @@ public function thatChannelAllowsToShopUsingAndCurrencies(ChannelInterface $chan
     public function itUsesTheCurrencyByDefault(ChannelInterface $channel, $currencyCode)
     {
         $currency = $this->provideCurrency($currencyCode);
-        $currency->setExchangeRate(1.0);
```

Comment:
This step is ``that channel uses the currency by default`` -> it should not set this currency's exchange rate to *1.0*, as exchange rate is not configured per channel, but for whole system. In my case, it caused a mistake when one channel has base currency ``EUR`` and the other one ``GBP``.